### PR TITLE
Support for animations

### PR DESCRIPTION
**Pull request by MarcoBuster**
* New `Animation` class
* New attribute `Message.animation` 
* New method `Chat.send_gif`
* New method `User.send_gif`
* New method `Message.reply_with_gif`

### CODE REVIEW

File: docs/api/telegram.rst [hunk 1]
```diff
@@ -360,6 +360,41 @@ about its business.
...
+      its *url*, or its Telegram *file_id*. Only one of these arguments must be passed.
+
+      You may optionally specify the *duration*, the *width* and the *height* of the GIF.
+      If the GIF track you're sending is in reply to another message,
+      set *reply_to* to the ID of the other :py:class:`~botogram.Message`.
+
+      The *attach* parameter allows you to attach extra things like
+      :ref:`buttons <buttons>` to the message.
+
+      The *notify* parameter is for defining if your message should trigger
```
Comment (DTRandom) [inline docs/api/telegram.rst]: It's better to say "is to define" than "is for defining"

File: docs/api/telegram.rst [hunk 2]
```diff
@@ -360,6 +360,41 @@ about its business.
...
+      a notification on the client side (yes by default).
+
+      :param str path: The path to the animation
+      :param str file_id: The Telegram *file_id* of the animation
+      :param int duration: The animation duration, in seconds
```
Comment (DTRandom) [inline docs/api/telegram.rst]: It's better to say "The animation length" than "The animation duration"

File: docs/api/telegram.rst (hunk 1 above)
Comment (MarcoBuster) [reply docs/api/telegram.rst]: This will be fixed in another PR, is a global problem

File: docs/api/telegram.rst (hunk 2 above)
Comment (MarcoBuster) [reply docs/api/telegram.rst]: This will be fixed in another PR, is a global problem

**Pull request merged**